docs/json/sonarr/cf/av1.json

```diff
@@ -9,7 +12,7 @@
       "negate": false,
       "required": true,
       "fields": {
-      "value": "\\bAV1\\b"
+        "value": "\\bAV1\\b"
```

Comment:
```suggestion
      "value": "\\bAV1\\b"
```

Reply:
@bakerboy448 @yammes08 i'll revert this, but i've noticed a lot of discrepancies in the CFs:

- this change we're commenting on. `value` is a key under `fields` and should be indented.
- formatting an array of objects with like [this](https://github.com/TRaSH-Guides/Guides/blob/0b90b4689f3783d428a562f273729bb9fe1a3859/docs/json/sonarr/cf/dd.json#L9) vs [this](https://github.com/TRaSH-Guides/Guides/blob/0b90b4689f3783d428a562f273729bb9fe1a3859/docs/json/sonarr/cf/ddplus-atmos.json#L9-L10)
- 2 vs 4 space indentation
- incorrect indentation such as https://github.com/TRaSH-Guides/Guides/blob/0b90b4689f3783d428a562f273729bb9fe1a3859/docs/json/sonarr/cf/cr.json#L2-L4

If you like, I could make another PR to resolve these or is this intended?

Reply:
The jsons should match whatever is expected directly from Starr with only the specific trash fields added

Reply:
I think we picked this up after the org merge, that there's a bigger piece of work to do to standardise CF json formatting.